src/axelrod_dojo/algorithms/genetic_algorithm.py

```diff
@@ -1,26 +1,32 @@
+import itertools
+import os
 from itertools import repeat
 from multiprocessing import Pool, cpu_count
 from operator import itemgetter
 from random import randrange
 from statistics import mean, pstdev
 
 import axelrod as axl
-
 from axelrod_dojo.utils import Outputer, PlayerInfo, score_params
 
 
 class Population(object):
     """Population class that implements the evolutionary algorithm."""
+
     def __init__(self, params_class, params_kwargs, size, objective, output_filename,
                  bottleneck=None, mutation_probability=.1, opponents=None,
                  processes=1, weights=None,
-                 sample_count=None, population=None):
+                 sample_count=None, population=None, print_output=True):
         self.params_class = params_class
         self.bottleneck = bottleneck
-
+        self.print_output = print_output
```

Comment:
Consider making `print_output` and argument to `evolve` instead of a instance variable.

Reply:
I was thinking that eventually it could propagate through as a sort of verbosity level (print each cycle (or every n cycles) or each game etc.). Moving this to a method param would also be harder to control over multiple generation cycles; in my opinion using a getter & setter would be the easiest way to control the output levels.

Reply:
Ok, I don't feel strongly about this. Regardless there's not need for a getter or setter.

Reply:
> I was thinking that eventually it could propagate through as a sort of verbosity level (print each cycle (or every n cycles) or each game etc.)

In general I find "coding for the future" introduces complexities that don't often end up being used but need to be maintained. I'd prefer this to follow @marcharper 's suggestion :+1: